src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * Add `enum` env var processor
  * Add `shuffle` env var processor
  * Add `resolve-env` option to `debug:config` command to display actual values of environment variables in dumped configuration
+ * Allow `ServiceSubscriberInterface::getSubscribedServices()` to return `SubscribedService[]`
```

Comment:
Include here or just the Contracts changelog?

Reply:
just in the contracts would be fine to me